<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>X2Weather</class>
 <widget class="QDialog" name="X2Weather">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>342</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>364</width>
    <height>342</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>364</width>
    <height>342</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Solo Cloudwatcher</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_12">
   <item>
    <widget class="X2Form" name="X2Form">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>99999</width>
       <height>99999</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>1</width>
       <height>1</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="pushButtonCancel">
      <property name="geometry">
       <rect>
        <x>136</x>
        <y>288</y>
        <width>81</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonOK">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>232</x>
        <y>288</y>
        <width>81</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>80</y>
        <width>305</width>
        <height>192</height>
       </rect>
      </property>
      <property name="title">
       <string>Status</string>
      </property>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>32</y>
         <width>144</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Ambient Temperature :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>72</x>
         <y>56</y>
         <width>88</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Humidity :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>72</x>
         <y>80</y>
         <width>88</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Dew Point :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="temperature">
       <property name="geometry">
        <rect>
         <x>176</x>
         <y>32</y>
         <width>112</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>-.- ºC</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="humidity">
       <property name="geometry">
        <rect>
         <x>176</x>
         <y>56</y>
         <width>112</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>-- %</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="dewPoint">
       <property name="geometry">
        <rect>
         <x>176</x>
         <y>80</y>
         <width>112</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>-.- ºC</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_23">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>104</y>
         <width>144</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Barometric Pressure :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="pressure">
       <property name="geometry">
        <rect>
         <x>176</x>
         <y>104</y>
         <width>112</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>----.- mbar</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>24</x>
         <y>136</y>
         <width>136</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Wind speed :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="windSpeed">
       <property name="geometry">
        <rect>
         <x>176</x>
         <y>136</y>
         <width>112</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>--.- km.h</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>56</x>
         <y>160</y>
         <width>104</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Wind gust :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="windGust">
       <property name="geometry">
        <rect>
         <x>176</x>
         <y>160</y>
         <width>112</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>--.- km.h</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>8</y>
        <width>304</width>
        <height>64</height>
       </rect>
      </property>
      <property name="title">
       <string>Connection settings</string>
      </property>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>32</y>
         <width>88</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>IP Address :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="IPAddress">
       <property name="geometry">
        <rect>
         <x>112</x>
         <y>32</y>
         <width>184</width>
         <height>22</height>
        </rect>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>X2Form</class>
   <extends>QFrame</extends>
   <header>x2form.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
